### ISSUE TITLE
How to provide mycli completions to other clients like Emacs?

### ISSUE THREAD

**Issue by stardiviner** [dbcli/mycli#593]
I want to know whether it is possible to provide mycli completions to other clients like Emacs? If it is possible, then can you give me some hints and point me to the right direction? I want to implement an Emacs client based on mycli. Take mycli/pgcli power to Emacs.
Thanks for your great work!!

**Comment by stardiviner** [comment]
@meeuw Not able to use mycli/pgcli as a daemon?

**Comment by meeuw** [comment]
No, not at this moment, sorry.  It might be worth investigating, is there a framework for Emacs for getting completions from a daemon?

**Comment by stardiviner** [comment]
Yes, Emacs has many packages get completion from external daemons, I can list some examples here.
- http://company-mode.github.com/
- https://github.com/nsf/gocode
- https://github.com/jorgenschaefer/elpy

**Comment by rolandwalker** [comment]
Unfortunately, providing an API to Mycli completions is not in scope for the project.
